frontend/src/components/DonationPicker.js

```diff
@@ -30,10 +30,10 @@ export default class DonationPicker extends Component {
   }
 
   presetListItem(presetLabel) {
-    const { amount, currency, i18n, range } = this.props;
+    const { amount, currency, i18n } = this.props;
     let amountLabel, amountValue;
     if (presetLabel === 'other') {
-      amountValue = range ? range[0] : 50;
```

Comment:
this removes default of 50. I don't know the flow well enough to know if that's still needed?